crmsh/bootstrap.py

```diff
@@ -1740,7 +1748,7 @@ def join_csync2(seed_host):
     # they haven't gone to all nodes in the cluster, which means a
     # subseqent join of another node can fail its sync of corosync.conf
     # when it updates expected_votes.  Grrr...
-    if not invoke('ssh -o StrictHostKeyChecking=no root@%s "csync2 -mr / ; csync2 -fr / ; csync2 -xv"' % (seed_host)):
+    if not invoke('ssh -o StrictHostKeyChecking=no root@{} "csync2 -mr /; csync2 -xv || csync2 -fr / && csync2 -xv"'.format(seed_host)):
```

Comment:
Just to keep the commands more aligned I would replace
- `-mr` by `-rm`
- `xv` by `rxv` (is there any reason why we don't put `r` here ?)
- `-fr` by `-rf`.
This way is easier to identify that the used commands are the same than in the other usages

Reply:
Changed and sync to other two branches